### ISSUE TITLE
Force cookies to be non-secure

### ISSUE THREAD

**Issue by maritaria** [antoinechalifour/memento#55]
The api I am talking to (and using memento as caching for) uses secure cookies meaning that on requests to memento these do not get transfered because memento is HTTP only.

This seems a simpler solution than supporting both HTTP and HTTPS on the same port

**Comment by antoinechalifour** [comment]
Cookies has been challenging since day 1. I'm not really sure how to deal with them, do you have opinions about these?

As your API and memento do not run on the same domain, I'm not sure cookies can be forwarded. Maybe Memento could set the cookies, changing the domain, and then forward them?

What do you think?

**Comment by maritaria** [comment]
Cookies work at least for first party because the domain is the same.
However when it comes to 3rd party domains it becomes tricky because your
browser is recognizing memento as localhost. Also something I noticed is
that an api which responds every request with an updated cookie will not
allow requests to be cached because those change the hash of future
requests.

**Comment by antoinechalifour** [comment]
Hi @maritaria ! I just opened #61 which should convert secure cookies to normal ones. Could you checkout the branch locally and test it very quick?

If you need an API which sets secure cookies, you can use:

```json
{
  "targetUrl": "https://memento-cookies-oxedhxgpmu.now.sh"
}
```

I'm not very sure about the implementation though.

**Comment by antoinechalifour** [comment]
I decided to change the implementation an a new implementation is available at #71 
Would be very nice if you someone could checkout the branch locally and test it :)
